src/jsp/com/cubrid/jsp/Server.java

```diff
@@ -74,6 +76,16 @@ private Server(
         ServerSocket serverSocket = null;
         int port_number = Integer.parseInt(port);
         try {
+            String logFilePath =
+                    rootPath
+                            + File.separatorChar
+                            + LOG_DIR
+                            + File.separatorChar
+                            + serverName
+                            + "_java.log";
+            loggingThread = new LoggingThread(logFilePath);
+            loggingThread.start();
+
```

Comment:
Is the file name determined only when it is first driven?
Isn't there a time when we need to make a new file like when it's over a certain size?

Reply:
The log file is no limit?

Reply:
As a specification that is already defined, the file name is determined. In logging.properties in the jsp folder, you can set an option to create a new log file when the log file grows to a certain size. This is not controlled in code.